### PR TITLE
fix(ZNTA-1169): Use different default username and password

### PR DESCRIPTION
**Pull request by definite**
Change the zanata.properties and documentation, in order to provide an working zanata installation instruction for overlay users.

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-server/1202)

### CODE REVIEW

**Comment by carlosmunoz** [comment]
Reviewed 1 of 2 files at r1.
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[docs/user-guide/system-admin/configuration/installation.md, line 24 \[r1\]](https://reviewable.io:443/reviews/zanata/zanata-server/1202#-KKBtK-wgxLJgBnP4Bez:-KKBtK-wgxLJgBnP4Bf-:-732395060) ([raw file](https://github.com/zanata/zanata-server/blob/55d749b0759cb0a31f24590394b34251e0b8ec91/docs/user-guide/system-admin/configuration/installation.md#L24)):*
> ```Markdown
>  
>  1. Grant `zanata` user the permission
>  `GRANT ALL ON zanata.* TO 'zanata'@'localhost'`
> ```

How about:

1. Create a `zanata` user and grant permissions

```
CREATE USER 'zanata'@'localhost' IDENTIFIED BY 'ChangeMe';
GRANT ALL ON zanata.* TO 'zanata'@'localhost';
```

---

*[docs/user-guide/system-admin/configuration/installation.md, line 27 \[r1\]](https://reviewable.io:443/reviews/zanata/zanata-server/1202#-KKBttRWA15bXhXIaCvs:-KKBttRXkr3K4WEGWwmV:1874798145) ([raw file](https://github.com/zanata/zanata-server/blob/55d749b0759cb0a31f24590394b34251e0b8ec91/docs/user-guide/system-admin/configuration/installation.md#L27)):*
> ```Markdown
> 
>  1. Start MySQL service and create a database schema for Zanata.
>  `CREATE DATABASE zanata /**!40100 DEFAULT CHARACTER SET utf8 **/`
> ```

I think this might be ok, but lets make sure the database creation can come after the user permissions.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1202)*

File: docs/user-guide/system-admin/configuration/installation.md [hunk 1]
```diff
@@ -16,9 +16,16 @@ The following packages are optional, but recommended:
 
  1. Download and install MySQL 5.5 from the [MySQL download page](http://dev.mysql.com/downloads/mysql/).
  Zanata has been thoroughly tested against MySQL 5.5 and the Zanata team therefore recommends that you install and use this version with Zanata.
+ 
+ 1. Create `zanata` user with the password `zanatapw`
+ `CREATE USER 'zanata'@'localhost' IDENTIFIED BY 'zanatapw'`
+ 
+ 1. Grant `zanata` user the permission
```
Comment (djansen-redhat) [inline docs/user-guide/system-admin/configuration/installation.md]: Grant `zanata` user full permissions

**Comment by carlosmunoz** [comment]
Reviewed 1 of 2 files at r2.
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[docs/user-guide/system-admin/configuration/installation.md, line 27 \[r1\]](https://reviewable.io:443/reviews/zanata/zanata-server/1202#-KKBttRWA15bXhXIaCvs:-KMRLjCb9htaZ2Yb3b6Z:-440910163) ([raw file](https://github.com/zanata/zanata-server/blob/55d749b0759cb0a31f24590394b34251e0b8ec91/docs/user-guide/system-admin/configuration/installation.md#L27)):*
<details><summary><i>Previously, carlosmunoz (Carlos Munoz) wrote…</i></summary>
> I think this might be ok, but lets make sure the database creation can come after the user permissions.
</details>
any thoughts on this?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1202)*

**Comment by definite** [comment]
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[docs/user-guide/system-admin/configuration/installation.md, line 27 \[r1\]](https://reviewable.io:443/reviews/zanata/zanata-server/1202#-KKBttRWA15bXhXIaCvs:-KNnzz6S3PgqhbzisfuZ:-93902811) ([raw file](https://github.com/zanata/zanata-server/blob/55d749b0759cb0a31f24590394b34251e0b8ec91/docs/user-guide/system-admin/configuration/installation.md#L27)):*
<details><summary><i>Previously, carlosmunoz (Carlos Munoz) wrote…</i></summary>
> any thoughts on this?
</details>
Yes, the database creation can definitely come after the user permission. You can drop the database and recreate it, the user permission still there.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1202)*

**Comment by carlosmunoz** [comment]
:lgtm:
-(Awaiting review) +Reviewed 

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-server/1202#-:-KNoiZprSAWhpGbUX20a:-1149335731)*

**Pull request merged**